.github/reaction.yml

```diff
@@ -0,0 +1,5 @@
+# Configuration for https://probot.github.io/apps/reaction/
+exemptLabels: []
+reactionComment: >
+  :wave: @{comment-author}, did you mean to use

```

Comment:
I think we should tweak the message a bit (_"did you mean to use"_ may sound a bit so/so (and I know people can get offended easily)).

Here's the text we used when we had "leeroy" running; https://github.com/docker/leeroy/blob/2fe5a91ab1f8a99baf64284abaae924687ed5570/github/issue.go#L13-L24

And/or we could link to the ["reporting other issues"](https://github.com/moby/moby/blob/7b3966245de27b753c6acfe0347833811ed9eed8/CONTRIBUTING.md#reporting-other-issues) paragraph (we may want to tweak that one a bit as well)